src/main/java/io/deephaven/csv/reading/ReaderUtil.java

```diff
@@ -0,0 +1,93 @@
+package io.deephaven.csv.reading;
+
+import io.deephaven.csv.containers.ByteSlice;
+import io.deephaven.csv.tokenization.RangeTests;
+import io.deephaven.csv.util.MutableInt;
+
+public class ReaderUtil {
+    public static String[] makeSyntheticHeaders(int numHeaders) {
```

Comment:
This can be moved / made private to the single callsite?

Reply:
Oops. There were supposed to be two call sites. Now there are :-).  Thanks, good catch.